### ISSUE TITLE
OZW Admin UnknownSocketError

### ISSUE THREAD

**Issue by A-Bomb** [home-assistant/hassio-addons#1494]
I have a strange issue where I cannot connect to the OZW Admin via embedded QT, TightVNC, or the OpenZwave desktop app.  I have a fresh HassOS 4.11 install on a Raspberry Pi 4 B.  Barely any add-ons and integrations (Check Config, File Editor, Samba, and Terminal).  I am using the official Mosquito Broker 5.1 add-on and integration and the OpenZWave add-on is 0.5.2.  I have uninstalled and installed Mosquito and OpenZwave multiple times with reboots and configs seem correct.  Everytime I try and start the OWZdaemon on 1983 in any interface, I get a progress dialog of ConnectingState of 0% that hangs and a UnknownSocketError in the log.

**When I connect via embedded QT client, this is the OpenZWave log:**
```
20200722 10:21:26.374 CDT] [ozw.library] [debug]: Detail - Node: 255 Removing current message 
172.30.32.2 - - [22/Jul/2020 10:22:14] 172.30.32.2: Plain non-SSL (ws://) WebSocket connection
172.30.32.2 - - [22/Jul/2020 10:22:14] 172.30.32.2: Path: '/websockify'
172.30.32.2 - - [22/Jul/2020 10:22:14] connecting to: 127.0.0.1:5900
[20200722 10:22:25.181 CDT] [ozwadmin] [debug]: Doing Remote Connection: QUrl("ws://localhost:1983") 
[20200722 10:22:25.185 CDT] [ozw.options] [debug]: Nothing to do till we connect 
[20200722 10:22:25.197 CDT] [ozw.manager] [debug]: Attempting Connection to  QUrl("ws://localhost:1983") 
[20200722 10:22:25.197 CDT] [ozw.manager] [info]: WebSocket Client State Changed:  QAbstractSocket::ConnectingState 
[20200722 10:22:25.197 CDT] [ozwadmin] [debug]: Remote Connection Status:  QTOZWManager::ConnectingState QAbstractSocket::UnknownSocketError 
[20200722 10:22:25.197 CDT] [ozwadmin] [debug]: QTOZWManager::ConnectingState QAbstractSocket::UnknownSocketError 
[10:22:33] INFO: Starting ozw-admin...
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
ozwadmin: Settings Path is at  "/root/.config/OpenZWave/ozw-admin.conf"
ozwadmin: Checking  "/data/ozw/config"  for manufacturer_specific.xml
ozwadmin: Checking  "/data/ozw/config"  for options.xml
ozwadmin: DBPath:  "/data/ozw/config/"
ozwadmin: userPath:  "/data/ozw/config/"
ozwadmin: Settings Path:  "/root/.config/OpenZWave/ozw-admin.conf"
ozwadmin: Using Saved Network Cache Size: 1000
QT_MESSAGE_PATTERN: %{backtrace} is not supported by this Qt build
[20200722 10:22:34.258 CDT] [ozwadmin] [info]: OZWCore Initilized QTOpenZwave(0x7fb66fd1e0) QTOZWManager(0x55860f5280) 
[20200722 10:22:34.320 CDT] [default] [warning]: QVncServer created on port 5900 
1595431372: Client qt-openzwave-1 has exceeded timeout, disconnecting.
```

**When I connect via desktop TightVNC Client, this is the OpenZWave log:**
```
[20200722 10:24:12.854 CDT] [ozwadmin] [debug]: Doing Remote Connection: QUrl("ws://localhost:1983") 
[20200722 10:24:12.859 CDT] [ozw.options] [debug]: Nothing to do till we connect 
[20200722 10:24:12.870 CDT] [ozw.manager] [debug]: Attempting Connection to  QUrl("ws://localhost:1983") 
[20200722 10:24:12.870 CDT] [ozw.manager] [info]: WebSocket Client State Changed:  QAbstractSocket::ConnectingState 
[20200722 10:24:12.870 CDT] [ozwadmin] [debug]: Remote Connection Status:  QTOZWManager::ConnectingState QAbstractSocket::UnknownSocketError 
[20200722 10:24:12.870 CDT] [ozwadmin] [debug]: QTOZWManager::ConnectingState QAbstractSocket::UnknownSocketError 
```

I can connect to other ports with SSH and MQTT Explorer ,so I am not sure why I am getting UnknownSocketError specifically on port 1983.  Any ideas on what I am missing here?   What's staring me in the face that I can't see?  Thanks in advance for taking time to look at this.

**Comment by TonyRoomZ** [comment]
I also am having this issue. I tried even exposing port 1983 and connecting with ozw-admin from a remote machine and see the same UnknownSocketError in the logs.  Hardware is also a fresh hassOS 64 bit image install on a rpi 4.

**Comment by A-Bomb** [comment]
Thanks! I will give it a try tonight and see if that is the issue.  You also gave me a couple  more ideas to test.

**Comment by A-Bomb** [comment]
OK my issue was in my OpenZWave config where I specified the USB Path as ```device: /dev/ttyACM0```.  All documentation I found for ZWave Sticks was to use this path in the config.  However I am using a Nortek USB ZWave & ZigBee Adapter (HUSBZB-1).  This actually creates two more USB Serial Port adapters ```/dev/ttyUSB0``` and ```/dev/ttyUSB1``` .  So it seems that ```/dev/ttyUSB0``` is the Zwave adapter and ```/dev/ttyUSB1``` is the ZigBee adapter.  So I used ```/dev/ttyUSB0``` in my OpenZWave config and all is good.  Thanks for the inspiration @TonyRoomZ to get me here :)
